torchci/pages/[repoOwner]/[repoName]/pull/[prNumber].tsx

```diff
@@ -92,6 +94,96 @@ function CommitHeader({
   );
 }
 
+function DrCIButton({
```

Comment:
Nit: Better to have this button as a component I think instead putting it here in the pr page